generators/server/templates/build.gradle.ejs

```diff
@@ -634,6 +634,12 @@ if (project.hasProperty("nodeInstall")) {
         download = true
     }
 }
+
+// Workaround npm failing with audit
+// https://github.com/jhipster/generator-jhipster/pull/14315#issuecomment-844149669
+task npmInstallNoAudit(type: NpmTask) {
```

Comment:
A bit easier could be to use the convention task

```
npm_install {
  args = ['--no-audit']
}
```

And let the other tasks depend on `npm_install` instead of `npmInstall`